lambda/custom/.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "semi": true,
+  "trailingComma": "none",
+  "singleQuote": true,
+  "tabWidth": 2
+}
```

Comment:
Ditto here -- I'd rather we stick to defaults unless there is some reason not to.